tests/lib/forms-schema.test.js

```diff
@@ -4201,3 +4267,23 @@ test('should allow placeholderValue property for these elements', () => {
   )
   expect(result.error).toBe(null)
 })
+
+test('should not allow forms without tags', () => {
```

Comment:
We still want to allow forms without tags, it should default to setting tags as an empty array. This way, we are not making a breaking change to our API and all forms will have an array for `tags`